docs/index.asciidoc

```diff
@@ -70,6 +91,22 @@ The IP address or hostname to listen on.
 
 The port to listen on. Remember that ports less than 1024 (privileged
 ports) may require root to use.
+port_tcp and port_udp can be used to set a specific port for each protocol.
+[id="plugins-{type}s-{plugin}-port_tcp"]
+===== `port_tcp` 
+
+  * Value type is <<number,number>>
+  * Default value is ``
```

Comment:
Let's use `There is no default value for this setting.` here instead

Reply:
Done